numpy/random/generator.pyx

```diff
@@ -3786,7 +3786,7 @@ cdef class Generator:
     # Shuffling and permutations:
     def shuffle(self, object x, axis=0):
         """
-        shuffle(x)
+        shuffle(x, axis=0)
```

Comment:
I'm surprised these are needed at all - does cython not work with `inspect.signature`?

Reply:
I haven't looked into it. I only noticed that the axis was missing when I used `rng.shuffle?` in ipython.  It shows only the docstring, with no separate line for the signature.

Reply:
Perhaps I'll take a look at this at some point